typing_tester.py

```diff
@@ -0,0 +1,173 @@
+import curses
+from curses import wrapper
+import time
+import requests
+
+def get_random_word():
+    """Get random word with pronunciation from RapidApi"""
+
+    url = "https://random-words-api-umber.vercel.app/word"
+
+    headers = {
+        "X-RapidAPI-Key": "f934e406b8msh16f9dee78137dfcp1fab91jsn560d9925327d",
+        "X-RapidAPI-Host": "random-words-with-pronunciation.p.rapidapi.com"
+    }
+
+    response = requests.get(url, headers=headers)
```

Comment:
Write complete docstring for this function since its calling external API. Make sure to add the sample response of the API call as well